netjsonconfig/backends/openvpn/parser.py

```diff
@@ -4,8 +4,8 @@
 from ...utils import sorted_dict
 from ..base.parser import BaseParser
 
-vpn_pattern = re.compile('^# openvpn config:\s', flags=re.MULTILINE)
-config_pattern = re.compile('^([^\s]*) ?(.*)$')
+vpn_pattern = re.compile(r'^# openvpn config:\s', flags=re.MULTILINE)
+config_pattern = re.compile(r'^([^\s]*) ?(.*)$')
```

Comment:
try not to mix changes that are unrelated to the problem you are fixing, or if you do, at least commit them separately with a commit message that indicates the change is a minor cleanup. For this time it's ok, keep in mind for the next patches.

Reply:
Okay, I'll keep that in mind! I've added this because it was causing errors in the build, but next time I'll make a separate PR for it.